src/client/sandbox/event/listeners.js

```diff
@@ -68,7 +68,7 @@ export default class Listeners extends EventEmitter {
     }
 
     _createEventHandler () {
-        var listeningCtx = this.listeningCtx;
+        var self = this;
```

Comment:
Why we need this change? And BTW, never use `self`, `_this` or anything like that for the closures. Use meaningful names instead. E.g. in this case:
```js
var listeners = this;
```